### PR TITLE
Blog: Hawkular Alerting Tutorial announcement

### PR DESCRIPTION
**Pull request by jshaughn**

### CODE REVIEW

**Comment by jshaughn** [comment]
@lucasponce or @pilhuhn , any issues?

File: src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+= Hawkular Alerting: Tutorial Now Available!
+Jay Shaughnessy
+2017-03-14
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerting
+
+Hawkular Alerting is a pluggable, API-driven component for adding alerting to your application or tool.  It's one of the many http://hawkular.org[Hawkular] offerings for monitoring and management.  To help quick-start new users, and as a reference for current users, we've released the first five lessons of a new tutorial, with more to follow.  It's a great way to hit the ground running.
+
+To get started just clone or download http://github.com/hawkular/hawkular-alerts[Hawkular Alerting from GitHub].  You'll find the tutorial under the `examples` directory. Start with the README.adoc.  If you don't have an asciidoc extension in your browser you can see the rendered README https://github.com/hawkular/hawkular-alerts/blob/master/examples/README.adoc[directly from GitHub].
```
Comment (lucasponce) [inline src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc]: Even if it is trivial, I like the style of copy/paste for ultra lazies.
So, some box code text with the commands ready to start people invites to test.
(In previous blogs is more or less how everyone suggests).

**Comment by lucasponce** [comment]
LGTM @jshaughn but I have not rights to merge on this repo.

**Comment by jshaughn** [comment]
@pilhuhn perhaps you can merge?

File: src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+git clone https://github.com/hawkular/hawkular-alerts.git
+cd hawkular-alerts/examples/tutorial
+----
+
+====== For Others:
+
+. Click this link to visit http://github.com/hawkular/hawkular-alerts[Hawkular Alerting on GitHub].
+. Click _Clone or download_
+. Click _Download ZIP_
+
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc:22]: Try to remove that empty line to see if the code below gets indended to the right level.

File: src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+----
+unzip hawkular-alerts-master.zip
+cd hawkular-alerts/examples/tutorial
+----
+
+Start with the README.adoc. If you don't have an asciidoc extension in your browser you can read the rendered README https://github.com/hawkular/hawkular-alerts/blob/master/examples/tutorial/README.adoc[directly from GitHub].  From here just click links in the document to proceed.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc]: Add a section heading before this to guide the reader that the "common" instructions continue here

File: src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc (hunk 2 above)
Comment (jshaughn) [reply src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc:22]: I tried too long to actually get that numbered list to indent. I gave up.  I just added one more section header, hopefully good enough...

Comment (pilhuhn) [reply src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc:22]: Have a look at https://github.com/hawkular/hawkular.github.io/commit/41d92e5b73850e7cac81c26b1b811d8b2398e68e

Comment (jshaughn) [reply src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc:22]: OK, so using the larger section heading did the trick.  Glad you found happiness :)

**Comment by pilhuhn** [comment]
On 17 Mar 2017, at 13:02, Jay Shaughnessy wrote:
> OK, so using the larger section heading did the trick.  Glad you found 
> happiness :)

No not the larger section headings, but the one line that is
not shown in the diff

https://github.com/hawkular/hawkular.github.io/commit/41d92e5b73850e7cac81c26b1b811d8b2398e68e#diff-b09e8a136e6eb9c36ee36caa3963680bR22

Larger section headings is the icing on the cake :)

-- 
hwr@pilhuhn.de  m:0179/207 4919  b:http://pilhuhn.blogspot.com

**Pull request merged**